<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimiterCalculator</class>
 <widget class="QMainWindow" name="LimiterCalculator">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="SpeakerInfoLabel">
          <property name="text">
           <string>Speaker characteristics</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="gridWidget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLineEdit" name="SpeakerImpedanceValue"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="SpeakerImpedanceLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Impedance (Ohm)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="SpeakerPowerValue"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="SpeakerPowerLabel">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Power consumption (W RMS)</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AmplificationInfoLabel">
          <property name="text">
           <string>Amplifier Characteristics</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="gridWidget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="AmpImpedanceLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Amp impedance (Ohm)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLineEdit" name="AmpImpedanceValue"/>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="SensitivityValue"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="SensitivityLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Sensitivity (Select the correct Units)</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="AmpPowerValue"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="AmpPowerLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Amp power (W RMS for the specific impedance)</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="SensitivityUnitCombo">
             <item>
              <property name="text">
               <string>Voltage</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="ProtectionLabel">
             <property name="text">
              <string>Protection (%)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QComboBox" name="ProtectionCombo">
             <item>
              <property name="text">
               <string>0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>25</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>75</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>90</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="0">
          <widget class="QLabel" name="AttackLabel">
           <property name="text">
            <string>Attack </string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="RMSThresholdValue">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="PakLimiterLabel">
           <property name="text">
            <string>Peak Limiter</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="RMSThresholdLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="RMSThresholdUnitCombo">
           <item>
            <property name="text">
             <string>dBu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>dBfs (for DCX2496 users)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="PeakThresholdValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="RMSLimiterLabel">
           <property name="text">
            <string>RMS Limiter</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="PeakThresholdLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Threshold</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="AttackUnitsLabel">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLabel" name="TImeSettingsLabel">
           <property name="text">
            <string>Time settings</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="PeakThresholdUnitCombo">
           <item>
            <property name="text">
             <string>dBu</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>dBfs (for DCX2496 users)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QLineEdit" name="AttackValue"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="ReleaseLabel">
           <property name="text">
            <string>Release</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="ReleaseUnitLabel">
           <property name="text">
            <string>ms</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QLineEdit" name="ReleaseValue"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>file</string>
    </property>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="loadFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>load file</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveMod">
   <property name="text">
    <string>Modulation maps</string>
   </property>
  </action>
  <action name="saveRaw">
   <property name="text">
    <string>Raw images</string>
   </property>
  </action>
  <action name="saveNorm">
   <property name="text">
    <string>Normalized mod maps</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
